hoot-core/src/main/cpp/hoot/core/algorithms/zindex/LongBox.cpp

```diff
@@ -72,7 +72,7 @@ bool LongBox::edgeOverlaps(LongBox b)
   bool result = false;
   for (uint i = 0; i < getMin().size(); i++)
   {
-    result = result || (b.getMin()[i] == getMax()[i]) || (b.getMin()[i] == b.getMin()[i]) || (b.getMax()[i] == b.getMax()[i]);
+    result = result || (b.getMin()[i] == getMax()[i]) || (getMin()[i] == b.getMin()[i]) || (getMax()[i] == b.getMax()[i]);
```

Comment:
What about b.getMax()[i] == getMin()[i] ?
I ask because I'm just looking for code symmetry.

Reply:
Good catch.  I updated it and made it easier to read.